src/index.ts

```diff
@@ -147,3 +151,10 @@ export async function isAvailable() {
     throw err;
   }
 }
+
+/**
+ * reset the memoized isAvailable() lookup.
+ */
+export function resetIsAvailableCache() {
+  cachedIsAvailableResponse = undefined;
```

Comment:
Maybe have one more test that verifies that `resetIsAvailableCache` actually resets the cache.

Reply:
👍 I added a test for this.